firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+KeyCommands.swift

```diff
@@ -82,12 +75,8 @@ extension BrowserViewController {
                                      method: .press,
                                      object: .keyCommand,
                                      extras: ["action": "go-back"])
-
-        guard let tab = tabManager.selectedTab, tab.canGoBack else { return }
-
-        if !contentContainer.hasHomepage {
-            tab.goBack()
-        }
+        store.dispatch(GeneralBrowserAction(windowUUID: windowUUID,
```

Comment:
Here too: Dispatching the action should only happen if the feature flag for toolbar refactor is turned on.